Source/ThirdParty/easy_profiler/profiler_gui/CMakeLists.txt

```diff
@@ -0,0 +1,67 @@
+#set(CMAKE_PREFIX_PATH f:/qt/5.5/5.6/msvc2013_64/lib/cmake)
+
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTORCC ON)
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+find_package(Qt5Widgets)
+
+if (NOT Qt5Widgets_FOUND)
```

Comment:
This is supposed to be ```if (Qt5Widgets_FOUND)```?  I think a crumb may have been introduced when testing the warning message.